vispy/testing/_runners.py

```diff
@@ -190,8 +190,15 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(fnames_str):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
```

Comment:
different. PEP8 is a Python-wide standard. Numpy/scipy have their own set of specifications for how docstrings should be formatted for consistency and proper `sphinx` (doc-building) purposes. There are just a few rules, so it shouldn't take too long to get them right.